From e0f3836149889b1db7cce76218c76efbf0347e0e Mon Sep 17 00:00:00 2001
From: Richard Zhu <Richard.Zhu@freescale.com>
Date: Thu, 18 Jun 2015 15:09:47 +0800
Subject: [PATCH] MLK-11160 pci: imx: enlarge the delay time after resume

some pcie pe devices maybe still failed resime back,
after pcie rc is resumed, when the pcie ep devices have
a heavy loading task.

Enlarge the delay time after imx6qp pcie rc resume back.

Signed-off-by: Richard Zhu <Richard.Zhu@freescale.com>
---
 drivers/pci/host/pci-imx6.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/host/pci-imx6.c b/drivers/pci/host/pci-imx6.c
index 59eced5..03234aa 100644
--- a/drivers/pci/host/pci-imx6.c
+++ b/drivers/pci/host/pci-imx6.c
@@ -1206,7 +1206,7 @@ static int pci_imx_resume_noirq(struct device *dev)
 		if (ret < 0)
 			pr_info("pcie link is down after resume back.\n");
 		/* Delay for a while, give ep some times to resume from D3 */
-		udelay(200);
+		udelay(2000);
 	} else {
 		/*
 		 * L2 can exit by 'reset' or Inband beacon (from remote EP)
-- 
1.8.0

